cms/admin/pageadmin.py

```diff
@@ -840,6 +852,26 @@ def copy_page(self, request, page_id, extra_context=None):
 
     @require_POST
     @transaction.atomic
+    def revert_to_live(self, request, page_id, language):
+        """
+        Resets the draft version of the page to match the live one
+        """
+        page = get_object_or_404(self.model, id=page_id)
+
+        # ensure user has permissions to publish this page
+        if not self.has_revert_to_live_permission(request, language, obj=page):
+            return HttpResponseForbidden(force_text(_("You do not have permission to revert this page.")))
+
+        page.revert_to_live(language)
+
+        messages.info(request, _('The page "%s" was successfully reverted.') % page)
```

Comment:
``"%s" was reverted to the live version.`` would be better:

* tells us *what* it was reverted to
* "page" and "successfully" are both redundant